src/components/SignUp/Privacy.tsx

```diff
@@ -1,48 +1,119 @@
 import React from 'react';
+import styled from 'styled-components';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import { Link } from 'react-router-dom';
-import yellow_blob from '../../images/yellow_blob_1.svg';
+import yellow_blob from '../../images/yellow_blob_2.svg';
 import Image from 'react-bootstrap/Image';
+import { fonts } from '../../theme/styleVars';
 
 const Privacy: React.FC<{
   setForm: any;
   formData: any;
 }> = props => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { setForm, formData } = props;
+  const { formData } = props;
 
   return (
     <Container>
       <Row>
         <Col lg="8">
-          <h3>Hi, {formData.basicsFirstName}!</h3>
-          <h1>LET'S TALK PRIVACY</h1>
-          <h2>Your privacy is our top concern. Always.</h2>
-          <p>
+          <TitleThree>Hi, {formData.basicsFirstName}!</TitleThree>
+          <Title>LET'S TALK PRIVACY</Title>
+          <Tagline>Your privacy is our top concern. Always.</Tagline>
+          <PrivacyPar>
             We take privacy seriously (seriously). We will never share, sell, or
             otherwise distribute your information to a third party. We also know
             that some of the information we’re asking for is personal, and for
```

Comment:
I wondering if you're speaking about the sass declarations. I have to use the weird quotes for the interpolation of the colors and fonts.

Reply:
@aminnoah no, I mean the single quote in the copy here, like in the word "we're" - it may be hard to see, but the apostrophe is a special character quote instead of a browser safe single quote

Reply:
I see it now